src/actions/guides/http_and_routing/link_generation.cr

```diff
@@ -59,7 +59,7 @@ class Guides::HttpAndRouting::LinkGeneration < GuideAction
     In a web page, an anchor is a an element you can use to take a user to a specific part of a page.
     For example, we use this to link to different parts of a guide page.
 
-    The `path` method takes an `anchor` argument.
+    The `with` method takes an `anchor` argument.
```

Comment:
Actually, both of these are true... :thinking:  I guess `with` is better here since we use it below, but you can pass `anchor` to the `path` method like here https://github.com/luckyframework/website/blob/2f11bb43ee55c332dca39b26074718c5c846ec7a/src/actions/guides/frontend/rendering_html.cr#L312